docs/guide/tutorial-4-more-about-controllers.md

```diff
@@ -534,20 +534,17 @@ Reload the world.
 > ```MATLAB
 > % initialize devices
 > ps = [];
-> ps_names = [
->   "ps0", "ps1", "ps2", "ps3",
->   "ps4", "ps5", "ps6", "ps7"
-> ]
+> ps_names = [ "ps0", "ps1", "ps2", "ps3", "ps4", "ps5", "ps6", "ps7" ];
 >
 > for i = 1:8
->   ps[i] = wb_robot_get_device(ps_names[i]);
->   wb_distance_sensor_enable(ps[i], TIME_STEP);
+>   ps(i) = wb_robot_get_device(convertStringsToChars(ps_names(i)));
+>   wb_distance_sensor_enable(ps(i), TIME_STEP);
 > end
 > ```
 After initialization of the devices, initialize the motors:
 > ```MATLAB
-> left_motor = wb_robot_get_device("left wheel motor");
-> right_motor = wb_robot_get_device("right wheel motor");
+> left_motor = wb_robot_get_device('left wheel motor');
+> right_motor = wb_robot_get_device('right wheel motor');
```

Comment:
Why are you sometimes using simple quote and sometimes using double quotes for string in Matlab?

Reply:
Because it's not the same. Double quotes are MATLAB strings while single quotes are char arrays. It turns out our API functions need char arrays, but it's more convenient to make an array of strings and convert them afterwards to char arrays....